### PR TITLE
Feature/ui components

### PR DESCRIPTION
**Pull request by JessBrownless**

### CODE REVIEW

File: src/styles/global.css [hunk 1]
```diff
@@ -49,8 +49,32 @@
...
 
+.cols-50 {
+  grid-template-columns: 6fr 4fr;
+}
+
+.cols-50-reverse {
+  grid-template-columns: 5fr 5fr;
+}
+
+lg:pr-6 lg:pr-4
```
Comment (sonnybooth-hexsu) [inline src/styles/global.css]: Random line

File: src/styles/global.css [hunk 2]
```diff
@@ -49,8 +49,32 @@
...
   }
+
+  .grid-cols-hero {
+    grid-template-columns: 0.65fr 1fr;
+  }
+  
+}
+
+.rows-fr-auto {
+  grid-template-rows: auto 1fr;
```
Comment (sonnybooth-hexsu) [inline src/styles/global.css]: Remove any classes not used lets keep it clean

File: src/pages/index.js [hunk 3]
```diff
@@ -223,123 +229,182 @@ const IndexPage = () => (
...
-            <div>
-              <p className="mb-4 text-gray-900">
-                Dr Yalda is an expert in identifying the treatments that will
-                help you look and feel your best. Her experience and artistic
-                eye in conjunction with using the most premium brands allows her
-                to provide her patients with satisfied, long lasting results:
+      <div className="cols-50 rows-fr-auto pl-4 pr-4 grid grid-cols-2 grid-rows-2 gap-4">
+        <div className="col-span-2 flex content-center justify-center md:col-span-1 md:row-span-2">
+          <img
+            src="/static/cover-1-4b42b6386c67ac1a1074e200b0daf0a4.jpg"
```
Comment (sonnybooth-hexsu) [inline src/pages/index.js]: Weird src?

File: src/pages/index.js [hunk 4]
```diff
@@ -223,123 +229,182 @@ const IndexPage = () => (
...
+            <i className="lowercase">natural</i> beauty.
+          </h3>
+          <p className="mb-4 text-gray-900 md:text-xs lg:text-sm">
+            Dr Yalda is an expert in identifying the treatments that will help
+            you look and feel your best. Her experience and artistic eye in
+            conjunction with using the most premium brands allows her to provide
+            her patients with satisfied, long lasting results:
+          </p>
+          <ul>
+            {/* <li className="pb-1 pt-1">
```
Comment (sonnybooth-hexsu) [inline src/pages/index.js]: Remove commented out code

File: src/pages/index.js [hunk 5]
```diff
@@ -223,123 +229,182 @@ const IndexPage = () => (
...
-            </div>
-          </div>
+            </li>
+            <li className="pb-1 pt-1">
+              {" "}
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                width="24"
+                height="24"
+                viewBox="0 0 24 24"
```
Comment (sonnybooth-hexsu) [inline src/pages/index.js]: Remove all this. save this in a text file on your laptop if you want it later

File: src/pages/index.js [hunk 6]
```diff
@@ -223,123 +229,182 @@ const IndexPage = () => (
...
         </div>
       </div>
     </div>
-    {/*  Cover 01 Left Title */}
+    {/* Platforma Layout 02 */}
+    <div className="container mx-auto pl-6 pr-6 pb-10 pt-20 pb-20">
+      <div className="cols-50-reverse pl-4 pr-4 grid grid-cols-2 grid-rows-2 gap-10 md:col-gap-5">
+        <div className="col-span-2 flex content-center justify-center md:pl-6 md:col-span-1 md:col-start-2 md:row-span-2">
+          <img
+            src="/static/cover-1-4b42b6386c67ac1a1074e200b0daf0a4.jpg"
```
Comment (sonnybooth-hexsu) [inline src/pages/index.js]: Weird src?

File: src/pages/index.js [hunk 7]
```diff
@@ -223,123 +229,182 @@ const IndexPage = () => (
...
+            alt="ALT"
```
Comment (sonnybooth-hexsu) [inline src/pages/index.js]: "ALT"?

File: src/pages/index.js [hunk 8]
```diff
@@ -223,123 +229,182 @@ const IndexPage = () => (
...
+    {/* Benefits */}
+    <div className="container mx-auto pr-6 pl-6 pt-16 pb-16  text-center">
       <div className="pl-4 pr-4">
-        <div className="md:flex  md:flex-wrap lg:flex-col lg:relative">
-          <div className="md:order-2 md:w-1/2 lg:pr-3">
-            <h3 className="uppercase text-4xl leading-tight tracking-tight mb-6 md:text-5xl">
-              From a registered, <i className="lowercase">medical</i> doctor,
-              and Mentor.
-            </h3>
+        <div className="grid gap-16 xl:grid-cols-3 md:pr-48 md:pl-48 lg:pl-56 lg:pr-56 xl:pr-16 xl:pl-16 xl:gap-32">
```
Comment (sonnybooth-hexsu) [inline src/pages/index.js]: mq order

File: src/pages/index.js [hunk 9]
```diff
@@ -223,123 +229,182 @@ const IndexPage = () => (
...
-          <div className="md:w-full md:order-1 lg:w-1/2">
-            <img
-              className="mb-6"
-              src="https://frosty-ardinghelli-04f7f2.netlify.com/static/home-3-e5af273814dfcb6ee426fd73f696247e.jpg"
-              alt="Dr Yalda pushing a client&lsquo;s hair back, as part of the consultation process."
-            />
+          <div className="flex flex-col items-center">
+            <Clock size="40" className="inline mb-4 text-gray-900" />
+            <h5 className="text-xl mb-1">Reduce the signs of ageing</h5>
+            <p className="">
```
Comment (sonnybooth-hexsu) [inline src/pages/index.js]: remove className=""

File: src/pages/index.js [hunk 10]
```diff
@@ -223,123 +229,182 @@ const IndexPage = () => (
...
-                to provide her patients with satisfied, long lasting results:
-              </p>
-              <button className="bg-black hover:bg-gray-900 text-white font-light py-6 px-12 inline">
-                Book a consultation
-              </button>
-            </div>
+          <div className="flex flex-col items-center">
+            <Clock size="40" className="inline mb-4 text-gray-900" />
+            <h5 className="text-xl mb-1">Reduce the signs of ageing</h5>
+            <p className="">
```
Comment (sonnybooth-hexsu) [inline src/pages/index.js]: remove className=""

**Review by sonnybooth-hexsu — CHANGES_REQUESTED** [review]
clean up

File: src/pages/index.js (hunk 3 above)
Comment (JessBrownless) [reply src/pages/index.js]: Confused

File: src/pages/index.js (hunk 4 above)
Comment (JessBrownless) [reply src/pages/index.js]: Can't see it thinkits gone

File: src/pages/index.js (hunk 5 above)
Comment (JessBrownless) [reply src/pages/index.js]: Can't see this either

File: src/pages/index.js (hunk 7 above)
Comment (JessBrownless) [reply src/pages/index.js]: EVEN THAT ISN'T THERE

**Pull request merged**